service/src/main/java/uk/gov/hmcts/reform/fpl/validators/HasEPOGroundsValidator.java

```diff
@@ -0,0 +1,39 @@
+package uk.gov.hmcts.reform.fpl.validators;
+
+import uk.gov.hmcts.reform.fpl.enums.OrderType;
+import uk.gov.hmcts.reform.fpl.model.CaseData;
+import uk.gov.hmcts.reform.fpl.validators.interfaces.HasEPOGrounds;
+
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+
+public class HasEPOGroundsValidator implements ConstraintValidator<HasEPOGrounds, CaseData> {
+    @Override
+    public void initialize(HasEPOGrounds constraintAnnotation) {
+
+    }
+
+    @Override
+    public boolean isValid(CaseData caseData, ConstraintValidatorContext constraintValidatorContext) {
+        constraintValidatorContext.buildConstraintViolationWithTemplate(constraintValidatorContext
+            .getDefaultConstraintMessageTemplate()).addPropertyNode("groundsForTheApplication")
+            .addConstraintViolation();
+
+        if (caseData.getOrders() != null && caseData.getOrders().getOrderType() != null) {
```

Comment:
that could be simplified - probably extract method use the StringUtils helper and avoid multiple levels of if else if else.